src/model/user.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
```

Comment:
How are you using this?

Reply:
That is in the javascript course, but I'm also going to remove it